src/client/components/Modal/Modal.css

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
```

Comment:
I think @TITHI1244 has done a PR removing the need for these to be imported. Is that correct @TITHI1244 ?

Reply:
Yes, that PR is ready now. So the imports won't be needed :)